lib/govspeak/editions_extractor.rb

```diff
@@ -0,0 +1,24 @@
+module Govspeak
+  class EditionsExtractor
```

Comment:
I feel as though this name doesn't quite capture the purpose of this class.

How about `UnresolvedAdminLinkExtractor`. By `Unresolved` I am trying to indicate that it only returns admin links for editions which have not been published yet, but there could be a better word for it.